core/arch/arm/include/mm/core_mmu.h

```diff
@@ -82,11 +82,11 @@ extern vaddr_t core_mmu_linear_map_end;
  * MEM_AREA_TEE_RAM:  teecore execution RAM (secure, reserved to TEEtz, unused)
  * MEM_AREA_TEE_COHERENT: teecore coherent RAM (secure, reserved to TEEtz)
  * MEM_AREA_TA_RAM:   Secure RAM where teecore loads/exec TA instances.
- * MEM_AREA_NS_SHM:   NonSecure shared RAM between NSec and TEEtz.
+ * MEM_AREA_NSEC_SHM: NonSecure shared RAM between NSec and TEEtz.
```

Comment:
Thanks.
I think we could also change the several occurences of `TEEtz` into `TEE`.